<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="30321" started-at="2017-08-21T11:21:26Z" finished-at="2017-08-21T11:21:56Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="30321" started-at="2017-08-21T11:21:26Z" finished-at="2017-08-21T11:21:56Z">
      <class name="Runner.TestSuite">
        <test-method status="PASS" signature="tearUp()[pri:0, instance:Runner.TestSuite@553f17c]" name="tearUp" is-config="true" duration-ms="4654" started-at="2017-08-21T16:51:26Z" finished-at="2017-08-21T16:51:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearUp -->
        <test-method status="FAIL" signature="runSuite()[pri:0, instance:Runner.TestSuite@553f17c]" name="runSuite" duration-ms="24049" started-at="2017-08-21T16:51:30Z" finished-at="2017-08-21T16:51:54Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at PageModel.searchProduct.selectproduct(searchProduct.java:45)
	at Runner.TestSuite.runSuite(TestSuite.java:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runSuite -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:Runner.TestSuite@553f17c]" name="tearDown" is-config="true" duration-ms="1606" started-at="2017-08-21T16:51:54Z" finished-at="2017-08-21T16:51:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Runner.TestSuite -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
